### PR TITLE
Field selector in views

### PR DESCRIPTION
**Pull request by qsm-odoo**
TASK: https://www.odoo.com/web#id=33821&view_type=form&model=project.task&menu_id=

Implement the field selector **field** (to be able to select a field chain as a field's value).

Also improve the field selector widget so that x2x fields can be selected (and not automatically followed).

### CODE REVIEW

**Comment by qsm-odoo** [comment]
Note: I did not use the field anywhere in the views yet because there is a problem where I tried to use it (ir.model.fields): no way to access the model name of the field in creation/edition.

File: odoo/addons/base/ir/ir_model_view.xml [hunk 1]
```diff
@@ -225,6 +225,7 @@
                                 <field name="name"/>
                                 <field name="field_description"/>
                                 <field name="model_id" attrs="{'readonly': [('state','!=', 'manual')]}"/>
+                                <field name="model" invisible="1"/>
```
Comment (qsm-odoo) [inline odoo/addons/base/ir/ir_model_view.xml]: This is empty in creation even after choosing a model (`model_id`), this is a problem at python level. I do not know if there is any other place where I can use this `widget="field"`.

**Comment by rdeodoo** [comment]
@qsm-odoo Task cancelled, can we close?

**Comment by qsm-odoo** [comment]
Sad, but I suppose so...

**Pull request closed without merging**